jobs/haproxy/spec

```diff
@@ -27,6 +27,9 @@ properties:
   ha_proxy.internal_only_domains:
     description: "Array of domains for internal-only apps/services (not hostnames for the apps/services)"
     default: []
+  ha_proxy.trusted_domain_cidrs: 
+    description: "Space separated trusted cidr blocks for internal_only_domains"
+    default: 0.0.0.0/0
```

Comment:
So if an existing user of internal_only_domains has a domain set, but no trusted_domain_cidrs, the internal only domain will now become a wide-open domain, since no traffic is not in 0.0.0.0/0, so no traffic gets rejected. Switching to 0.0.0.0/32 would flip that, to put a secure default in place, such that if you have an internal_only_domain without any trusted_domain_cidrs, nothing can access the domain.

> On Jun 30, 2016, at 12:08 PM, sreenani <notifications@github.com> wrote:
> 
> In jobs/haproxy/spec <https://github.com/cloudfoundry-community/cf-haproxy-boshrelease/pull/25#discussion_r69162162>:
> 
> > @@ -27,6 +27,9 @@ properties:
> >    ha_proxy.internal_only_domains:
> >      description: "Array of domains for internal-only apps/services (not hostnames for the apps/services)"
> >      default: []
> > +  ha_proxy.trusted_domain_cidrs: 
> > +    description: "Space separated trusted cidr blocks for internal_only_domains"
> > +    default: 0.0.0.0/0
> 0.0.0.0/0 is allowing all source ip's by default.
> http-request deny if internal !private
> 
> deny if internal domain and if source is not trusted_domain_cidrs.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/cloudfoundry-community/cf-haproxy-boshrelease/pull/25/files/6156cc6105b859712befcdd859abb26fd92cdd81#r69162162>, or mute the thread <https://github.com/notifications/unsubscribe/AG2SUjuZ04-mXoPkOzPv2UnMU3nmvDCgks5qQ-oVgaJpZM4JBX8d>.
>